.circleci/config.yml

```diff
@@ -38,6 +38,7 @@ jobs:
             venv/bin/pip install --upgrade -r requirements.txt
             rm -rf sql/
             venv/bin/python script/generate_sql
+            venv/bin/python script/generate_views
```

Comment:
This invocation requires credentials in order to call out to BQ, so the test is currently failing.

I think I'm against adding this here, since it's going to potentially make valid PRs unrelated to view generation fail because a new table has been added.